src/main/resources/swagger/api-docs.yaml

```diff
@@ -53,6 +53,39 @@ paths:
       operationId: ping
       tags:
         - test
+  '/api/clusters':
+    get:
+      summary: List all clusters
+      description: List all clusters, optionally filtering on a set of labels
+      operationId: listClusters
+      tags:
+        - cluster
+      parameters:
+        - in: query
+          name: _labels
```

Comment:
Apparently swagger doesn't support free-form query string parameters. I.e. there is no way to specify this query string in swagger (where both the keys and values are dynamic):
```
http://leo/api/clusters?key1=val1&key2=val2&...
```
So I added the ability to specify labels in the following way so it can be done in swagger:
```
http://leo/api/clusters?_labels=key1%3Dval1,key2%3Dval2,...
```
Either approach works in Leo.

Reply:
Eeeeeww. What the hell, Swagger.

Can you add some more prose in the description pointing out the recommended way to use this endpoint? I imagine most users hit "Try it out" first and copy the format from there; it'd be good to explicate the _actually_ recommended way that we suggest API developers use.

Reply:
Sure, can do. Alternatively we could just leave out label filtering in Swagger, but I was thinking it would be bad to have hidden functionality in Leo not available in Swagger.

Reply:
I'm unhappy with having to code around Swagger's limitations but I'd be far more unhappy with undocumented endpoints. So this way is the right way around.